src/unix/async.c

```diff
@@ -38,6 +38,33 @@
 #include <sys/eventfd.h>
 #endif
 
+#if UV__KQUEUE_EVFILT_USER
+static uv_once_t kqueue_runtime_detection_guard = UV_ONCE_INIT;
+static int kqueue_evfilt_user_support = 1;
+
+
+static void uv__kqueue_runtime_detection(void) {
+  int kq;
+  struct kevent ev[2];
+  struct timespec timeout = {1, 0};
```

Comment:
why is this a 1 second sleep?
```suggestion
  struct timespec timeout = {0, 0};
```